lib/modules/manager/pip-compile/readme.md

```diff
@@ -17,15 +17,18 @@ You can "activate" the manager by specifying a `fileMatch` pattern such as:
 }
 ```
 
+`pip-compile` reads the output files to extract the arguments passed to the original command, as such the `fileMatch` must be configured for `*.txt` files and not `*.in`.
+
 ### Assumption of header with a command
 
-If Renovate matches a `pip-compile` output file it will extract original command that was used to create it from header in this file. Because of that `pip-compile` manager poses restrictions on how this file is generated:
+As Renovate matches a `pip-compile` output file it will extract original command that was used to create it from header in this file.
+Because of that `pip-compile` manager poses restrictions on how this file is generated:
 
 - Use default header generation, don't use `--no-header` option.
 - Pass all source files explicitly.
 
 In turn `pip-compile` manager will find all source files and parse them as package files.
-Currently only `*.in` files associated with `pip_requirements` manager and `setup.py` files associated with `pip_setup` manager are handled.
+Currently only `*.txt` files associated with `pip_requirements` manager and `setup.py` files associated with `pip_setup` manager are handled.
```

Comment:
This line may be reworded. But it was correct. I was referring here to these lines:

https://github.com/renovatebot/renovate/blob/752d2520fd3a354a4c59bb5efd0ed800156940f0/lib/modules/manager/pip-compile/extract.ts#L32-L34

Reply:
Oh, now I understand that sentence properly. But referencing the code you sent, wouldn't also `setup.cfg` and `pyproject.toml` be supported?

I'll try to rephrase the sentence, and add a list showing the matching of source files to the corresponding manager

Reply:
`setup.cfg` and `pyproject.toml` are detected but not handled. It should be easy enough to implement them. I don't use them myself and don't have time to test them.